src/HttpHandler/HttpHandlerFactory.php

```diff
@@ -45,6 +51,12 @@ public static function build(ClientInterface $client = null)
             $client = new Client(['handler' => $stack]);
         }
 
+        if ($logger === false) {
+            $logger = null;
+        } else {
+            $logger = $logger ?? ApplicationDefaultCredentials::getDefaultLogger();
+        }
```

Comment:
This is 100% not important, if you don't like it please disregard. I only suggest it because it sets the variable in one statement, so it _might_ be considered a little easier to follow (but that's entirely subjective)
```suggestion
        $logger = ($logger === false) 
            ? null
            : $logger ?? ApplicationDefaultCredentials::getDefaultLogger();
```

Reply:
I personally don't love the `??` inside a ternary, but I can see the appeal.